docs/resources/samples/excel-cross-reference.md

```diff
@@ -123,28 +110,58 @@ function main(workbook: ExcelScript.Workbook, keys: string): string {
+    returnString = "Event potentially overbooked. Please review."
   }
+
   console.log("Returning: " + returnString);
   return returnString;
 }
 
 // An interface representing a row of event data.
 interface EventData {
-  event: string
+  eventId: string
   date: number
   location: string
   capacity: number
 }
 ```
 
-## Training video: Cross-reference and format an Excel file
+## Power Automate flow: Check for inconsistencies across the workbooks
+
+This flow extracts the event information from the first workbook and uses that data to validate the second workbook.
+
+1. Create a new **Instant cloud flow**.
+1. Select **Manually trigger a flow** and press **Create**.
+1. Add a **New step** that uses the **Excel Online (Business)** connector with the **Run script** action. Use the following values for the action:
+    * **Location**: OneDrive for Business
+    * **Document Library**: OneDrive
+    * **File**: event-data.xlsx ([selected with the file chooser](../../testing/power-automate-troubleshooting.md#select-workbooks-with-the-file-browser-control))
+    * **Script**: Get event data
+
+    :::image type="content" source="../../images/cross-reference-flow-1.png" alt-text="The completed Excel Online (Business) connector for the first script in Power Automate.":::
+
+1. Add a second **New step** that uses the **Excel Online (Business)** connector with the **Run script** action. Use the following values for the action:
+    * **Location**: OneDrive for Business
+    * **Document Library**: OneDrive
+    * **File**: speaker-registration.xlsx ([selected with the file chooser](../../testing/power-automate-troubleshooting.md#select-workbooks-with-the-file-browser-control))
+    * **Script**: Validate speaker registration
+
+    :::image type="content" source="../../images/cross-reference-flow-2.png" alt-text="The completed Excel Online (Business) connector for the second script in Power Automate.":::
+1. This sample uses Outlook as the email client. You could use any email connector Power Automate supports. Add a **New step** that uses the **Office 365 Outlook** connector and the **Send and email (V2)** action. Use the following values for the action:
+    * **To**: Your test email account (or personal email)
+    * **Subject**: Event validation results
+    * **Body**: result (_dynamic content from **Run script 2**_)
 
-[Watch Sudhi Ramamurthy walk through this sample on YouTube](https://youtu.be/dVwqBf483qo").
+    :::image type="content" source="../../images/cross-reference-flow-3.png" alt-text="The completed Office 365 Outlook connector in Power Automate.":::
+1. Save the flow and try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations** and rows in **event-data**. Open **speaker-registrations** to see several highlighted cells where there are potential problems with the speaker registration listings.
```

Comment:
Add file extensions? 
```suggestion
1. Save the flow and try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations.xlsx** and rows in **event-data.xlsx**. Open **speaker-registrations.xlsx** to see several highlighted cells where there are potential problems with the speaker registration listings.
```

Reply:
Agreed, though I'm rephrase the try it out bit.